src/NovelRT/LoggingService.cpp

```diff
@@ -5,11 +5,17 @@
 
 namespace NovelRT {
 
+#ifndef NDEBUG
+  typedef spdlog::synchronous_factory spdlog_factory;
+#else
+  typedef spdlog::async_factory spdlog_factory;
+#endif
```

Comment:
Is this the only way to do this? Could we not use an `std::conditional` or is that not doable?

Reply:
I didn't try. This is keyed off a define, so I went with the same approach we use elsewhere

Reply:
OK, fair enough.